CHANGELOG.md

```diff
@@ -2,15 +2,37 @@
 
 ## Pending
 
-- (`ark-poly`) Reduce the number of field multiplications performed by `SparseMultilinearExtension::evaluate` and `DenseMultilinearExtension::evaluate`
+### Breaking changes
+
+### Features
+
+### Improvements
+
+### Bugfixes
+
+## v0.4.2
```

Comment:
So, you want a v0.4.3 that fixes the warnings? Should that be added here with a description of changes?